index.html

```diff
@@ -49,121 +49,203 @@ <h2>My Recent Works</h2>
+                        <h3 class="framework-text">Language</h3>
+                    </div>
+                    <ul class="tools-lists">
+                        <li>Javascript</li>
+                        <li>Ruby</li>
+                        <li>Html</li>
+                        <li>Css</li>
+                    </ul>
+                </div>
+                <div class="frame">
+                    <div class="language-icon">
+                        <div class="rectangle"></div>
+                        <!-- <img src="./images/square.png" alt="icon"> -->
+                        <h3 class="framework-text">Frameworks</h3>
+                    </div>
+                    <ul class="tools-lists">
+                        <li>Bootstrap</li>
+                        <li>Ruby on Rails</li>
+                        <li>RSPec</li>
+                        <li>CapyBara</li>
+                        <li>Selenium</li>
+                    </ul>
+                </div>
+                <div class="frame">
+                    <div class="language-icon">
+                        <div class="round"></div>
+                        <!-- <img src="./images/circle.png" alt="icon"> -->
+                        <h3 class="framework-text">Skills</h3>
+                    </div>
+                    <ul class="tools-lists">
+                        <li>Codekit</li>
+                        <li>Github</li>
+                        <li>Codepen</li>
+                        <li>Gitlab</li>
+                        <li>Terminal</li>
+                    </ul>
+                </div>
+
+            </div>
         </section>
-        <section class="languages">
-            <div class="language-icon">
-                <img src="./images/diamond.png" alt="icon">
-                <h3 class="framework-text">Language</h3>
-            </div>
-            <ul class="tools-lists">
-                <li>Javascript</li>
-                <li>Ruby</li>
-                <li>Html</li>
-                <li>Css</li>
-            </ul>
-            <div class="language-icon">
-                <img src="./images/square.png" alt="icon">
-                <h3 class="framework-text">Frameworks</h3>
-            </div>
-            <ul class="tools-lists">
-                <li>Bootstrap</li>
-                <li>Ruby on Rails</li>
-                <li>RSPec</li>
-                <li>CapyBara</li>
-                <li>Selenium</li>
-            </ul>
-            <div class="language-icon">
-                <img src="./images/circle.png" alt="icon">
-                <h3 class="framework-text">Skills</h3>
-            </div>
-            <ul class="tools-lists">
-                <li>Codekit</li>
-                <li>Github</li>
-                <li>Codepen</li>
-                <li>Gitlab</li>
-                <li>Terminal</li>
-            </ul>
-        </section>
-        <section>
+
+        <section class="website-form">
```

Comment:
- [ ] As a whole, I believe that this screencast will guide you on how your page looks when compared with Figma's design. Kindly remember that your work should be as close as possible to the figma design. Here is the video:

[Screencast from 05-03-2023 00:39:52.webm](https://user-images.githubusercontent.com/99715640/222934697-a6d536c3-efa8-4880-8789-966fee57055c.webm)

- Here is also an attached Figma design (Annotated version):
  - [ ] Kindly note how the sections have equal widths
  - [ ] How the images are all clearly visible
  - [ ] How the sections are not collapsing with both sides of the available viewport
  - [ ] How the background images are showing well
  - [ ] The stunning use of `margin` and `padding` for spacing.

![image](https://user-images.githubusercontent.com/99715640/222934803-30c28de6-5857-4e24-a2e9-56a0a5860dfd.png)